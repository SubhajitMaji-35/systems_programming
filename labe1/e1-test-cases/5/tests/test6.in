Everest
K2
Kangchenjunga
Lhotse
Makalu
Denali
MontBlanc
Annapurna
Matterhorn
Fuji
