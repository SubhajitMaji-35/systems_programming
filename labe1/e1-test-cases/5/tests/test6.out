veest
2
anchenunga
hose
aklu
enli
onBlan
nnpurn
aterhon
uj
